### PR TITLE
Release 1.9.0

### LINKED ISSUE #31: Create copy recipients using the SDK.

### PR DESCRIPTION
**Pull request by ahmadnazir**
Fixes #31

### CODE REVIEW

File: docs/recipient.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Recipients
```
Comment (janflora) [inline docs/recipient.md]: We call this a copy recipient everywhere else so I think we should be consistent.

**Review by janflora — CHANGES_REQUESTED** [review]
Substitute recipient -> copyRecipient and we are good to go.

File: docs/recipient.md (hunk 1 above)
Comment (ahmadnazir) [reply docs/recipient.md]: @janflora done

**Comment by janflora** [comment]
The changes were approved by me.

**Pull request merged**